server/helpers.js

```diff
@@ -188,15 +188,16 @@ module.exports = {
       });
   },
   getAnswers: (req, res) => {
+    // console.log(req.params)
     const id = req.params.question_id;
     const options = {
       headers: { Authorization: TOKEN },
       baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo',
       url: `/qa/questions/${id}/answers`,
       method: 'get',
       params: {
-        page: 1,
-        count: 5,
+        page: req.query.page,
+        count: req.query.count,
```

Comment:
Page and count should default to 1 and 5, so try
`page: req.query.page || 1,`
`count: req.query.count || 5,`
instead.